docker-compose.yml

```diff
@@ -139,8 +139,8 @@ services:
       - DISPLAY=${DISPLAY}
       - MUJOCO_GL=osmesa
       - XAUTHORITY=/root/.Xauthority
-    command: python -u -m roboai.roboai
-    # command:  /bin/bash -c "while true; do sleep 30; done"
+    # command: /bin/bash -c "python -u -m roboai.parse_images"
```

Comment:
Can you extend the `roboai` service, such as here: https://github.com/AshisGhosh/roboai/blob/0cc822570fb53ec01498e52c09634016207d1f77/docker-compose.yml#L87C3-L108C1